internal/testutil/unit/main_test.go

```diff
@@ -0,0 +1,14 @@
+package unit_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/unit"
+)
+
+func TestMain(m *testing.M) {
+	unit.InitializeAPISpecPaths()
```

Comment:
is InitializeAPISpecPaths being called twice, from TestMain and init?

Reply:
Removed in 41e4fb989e98a4f4f5f5da90d296d3195647d204